qa/L0_grpc/test.sh

```diff
@@ -123,16 +130,18 @@ else
     SIMPLE_IMAGE_CLIENT=../clients/image_client
     # SIMPLE_ENSEMBLE_IMAGE_CLIENT=../clients/ensemble_image_client
     SIMPLE_REUSE_INFER_OBJECTS_CLIENT=../clients/reuse_infer_objects_client
+    SIMPLE_KEEPALIVE_CLIENT=../clients/simple_grpc_keepalive_client
+    SIMPLE_CUSTOM_ARGS_CLIENT=../clients/simple_grpc_custom_args_client
     CC_UNIT_TEST=../clients/cc_client_test
 fi
 
-# Add string_dyna_sequence model to repo
-cp -r ${MODELDIR}/simple_dyna_sequence ${MODELDIR}/simple_string_dyna_sequence
-sed -i "s/simple_dyna_sequence/simple_string_dyna_sequence/g" ${MODELDIR}/simple_string_dyna_sequence/config.pbtxt
-sed -i "s/^platform: .*/backend: \"dyna_sequence\"/g" ${MODELDIR}/simple_string_dyna_sequence/config.pbtxt
-sed -i "/CONTROL_SEQUENCE_CORRID/{n;s/data_type:.*/data_type: TYPE_STRING/}" ${MODELDIR}/simple_string_dyna_sequence/config.pbtxt
-rm -f ${MODELDIR}/simple_string_dyna_sequence/1/model.graphdef
-cp ../custom_models/custom_dyna_sequence_int32/1/libtriton_dyna_sequence.so ${MODELDIR}/simple_string_dyna_sequence/1/
+## Add string_dyna_sequence model to repo
```

Comment:
Why was this commented? May be you added these changes into PR by mistake.

Reply:
Yes these were local changes added by mistake 👍